RebelAllianceBank/utils/FileHandler.cs

```diff
@@ -59,31 +75,33 @@ public static List<T> ReadFromFile<T>(string filePath, Func<string[], T> StoredD
         /// <summary>
         /// Converts a string array to an <see cref="IUser"/> object.
         /// </summary>
-        /// <param name="row">A string array representing a user row from the file.</param>
+        /// <param name="dataParts">A string array representing a user row from the file.</param>
         /// <returns>An <see cref="IUser"/> object or null if the row is invalid.</returns>
-        public IUser StoredUser(string[] row)
+        public IUser ParseUserFromData(string[] dataParts)
         {
-            if (row.Length == 6)
+            // ensure that the length is equal 6
+            if (dataParts.Length == 6)
             {
-                switch (row[4])
+                // depending on if the user is admin or not
+                switch (dataParts[4])
                 {
                     case "true":
                         return new Admin
                         {
-                            PersonalNum = row[0], // 8802252525
-                            Password = row[1],
-                            Surname = row[2],
-                            Forename = row[3],
-                            LoginLock = bool.Parse(row[5])
+                            PersonalNum = dataParts[0], // 8802252525
```

Comment:
88022525252 - är detta bara debug data eller något man behöver ha med sig vidare i koden?

Reply:
Just an old comment to explain how the PersonalNum should be read from file.